src/index.js

```diff
@@ -1,50 +1,49 @@
-const express = require("express");
-const cors = require("cors");
-const morgan = require("morgan");
-const low = require("lowdb");
-const swaggerUI = require("swagger-ui-express");
-const swaggerJsDoc = require("swagger-jsdoc");
-const busesRouter = require("./api/routes/buses.routes");
+const express = require('express')
+const cors = require('cors')
+const morgan = require('morgan')
+const low = require('lowdb')
+const swaggerUI = require('swagger-ui-express')
+const swaggerJsDoc = require('swagger-jsdoc')
+const busesRouter = require('./api/routes/buses.routes')
+const { PORT } = require('./config/dotenv')
```

Comment:
configure es6 and use it

Reply:
> configure es6 and use it

We can create a separate chore to handle that, it's completely different from this chore requirements